### DISCUSSION TITLE
Access to the source code of the new UI: https://app.tangoapp.dev/

### DISCUSSION THREAD

**Discussion by wbwxshao** [yume-chan/ya-webadb#609] [General]
Hello!
Thanks for creating this amazing library and this is exactly what we are looking for!
I saw the old-demo is open-sourced but I was really wondering if you could please also open source the code for the new demo app: https://app.tangoapp.dev/
Thanks in advance!

**Comment by yume-chan** [comment]
See #576 (Not Open Source section). If you want to know how some feature works, I can explain to you.

**Reply by wbwxshao** [reply]
Hi @yume-chan , I would like to have a private conversation with you, what is a good means to reach out to you and talk about this?
